src/views/Generator/TestOptions/LoadZones/LoadZoneRow.tsx

```diff
@@ -39,21 +39,23 @@ export function LoadZoneRow({ field, index, remove }: LoadZoneRowProps) {
   return (
     <Table.Row key={field.id}>
       <Table.Cell>
-        <FieldGroup errors={errors} name={`loadZones.${index}.loadZone`} mb="0">
+        <FieldGroup errors={errors} name={`zones.${index}.loadZone`} mb="0">
           <ControlledSelect
             control={control}
-            name={`loadZones.${index}.loadZone`}
+            name={`zones.${index}.loadZone`}
             options={getLoadZoneOptions()}
           />
         </FieldGroup>
       </Table.Cell>
       <Table.Cell>
-        <FieldGroup errors={errors} name={`loadZones.${index}.percent`} mb="0">
+        <FieldGroup errors={errors} name={`zones.${index}.percent`} mb="0">
           <TextField.Root
             type="number"
+            min="1"
+            max="100"
```

Comment:
I also included @going-confetti's suggestion from the previous PR to prevent values negative values when scrolling the mouse over the input box.